content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -258,6 +264,11 @@ You may use the `gen-jwt`
 [python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
 to generate a JWT with other list-typed claims for testing purposes.
 Follow the instructions in the `gen-jwt` script to use the `gen-jwt.py` file.
+To get the `key.pem` file, run the following command:
+
+{{< text bash >}}
```

Comment:
Please indent this code block with 4 spaces to ensure the numbered list doesn't break.